static/app/utils/replays/hooks/useCrumbHandlers.tsx

```diff
@@ -70,18 +69,15 @@ function useCrumbHandlers(startTimestampMs: number = 0) {
     [setCurrentHoverTime, removeHighlight]
   );
 
-  const handleClick = useCallback(
-    (frame: ReplayFrame) => {
-      setCurrentTime(frame.offsetMs);
-      setActiveTab(getFrameDetails(frame).tabKey);
-    },
-    [setCurrentTime, setActiveTab]
+  const onClickTimestamp = useCallback(
+    (frame: ReplayFrame) => setCurrentTime(frame.offsetMs),
+    [setCurrentTime]
   );
 
   return {
     handleMouseEnter,
     handleMouseLeave,
-    handleClick,
+    onClickTimestamp,
```

Comment:
The inconsistency here (and in `splitCrumb) with on vs handle is a bit confusing.